backend/app.ts

```diff
@@ -1,36 +1,58 @@
-import express, { Request, Response, NextFunction } from 'express';
-import dotenv from 'dotenv';
+import express, { Request, Response, NextFunction } from "express";
+import dotenv from "dotenv";
 dotenv.config();
-import routes from './routes';
-import connectDb from './services/database.service';
-
+import routes from "./routes";
+import connectDb from "./services/database.service";
+const cors = require("cors");
```

Comment:
Why do we need this syntax here? Could we also use import syntax like in previous cases?